README.md

```diff
@@ -24,8 +24,11 @@ Old Installation HowTo
 ===============
 Installation Ubuntu
 -------------------
-Use install_vrpn.sh to download, compile, and install VRPN.
-Please consult the VRPN website if your run into some trouble.
+A catkinized version of VRPN can be found here: https://github.com/ethz-asl/vrpn_catkin
+
+Alternative:
+ Use install_vrpn.sh to download, compile, and install VRPN.
+ Please consult the VRPN website if your run into some trouble.
```

Comment:
`if you run` ... not `if your run` and maybe place a link the to the website.

Personally, I would write something like:
`For further information about VRPN, please consult there website on: ...` or something like this.